schemas/reservationSchema.js

```diff
@@ -0,0 +1,69 @@
+const Joi = require('joi')
+const mongoose = require('mongoose')
+
+const reservationSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Types.ObjectId,
+    ref: 'User',
+    required: true,
+  },
+  hotelId: {
+    type: mongoose.Types.ObjectId,
+    ref: 'Hotel',
+    required: true,
+  },
+  roomlId: {
+    type: mongoose.Types.ObjectId,
+    ref: 'Room',
+    required: true,
+  },
+  startDate: {
+    type: Date,
+    required: true,
+  },
+  endDate: {
+    type: Date,
+    required: true,
+  },
+  people: {
+    type: {
```

Comment:
is it right? I mean type should be here?

Reply:
you can remove type, just pass adults and children
or create new schema 'whatevername' with adults and children, and make people: 'whatevername'